.github/pull_request_template.md

```diff
@@ -17,6 +17,11 @@
 2.
 3.
 
+### Compliance
+- [ ] My change is compliant with [ODFF Standard](https://docs.oasis-open.org/office/OpenDocument/v1.3/os/part4-formula/OpenDocument-v1.3-os-part4-formula.html).
+- [ ] My change is compatible with Microsoft Excel.
+- [ ] My change is compatible with Google Sheets.
```

Comment:
What do you think about putting the above points at the beginning of the `Checklist` below?

Reply:
Ok, I merged the `Compliance` and `Checklist` sections. Both ways are ok for me.